projects/make-a-vis/src/lib/visualization-view/visualization-view.module.ts

```diff
@@ -1,29 +1,33 @@
 // refer https://angular.io/guide/styleguide#style-03-06 for import line spacing
-import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
-// Material
+import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { MatButtonModule } from '@angular/material/button';
 import { MatChipsModule } from '@angular/material/chips';
 import { MatExpansionModule } from '@angular/material/expansion';
 import { MatIconModule } from '@angular/material/icon';
 import { MatInputModule } from '@angular/material/input';
 import { MatMenuModule } from '@angular/material/menu';
 import { MatTabsModule } from '@angular/material/tabs';
-
 import { DvlFwModule } from '@dvl-fw/core';
-// Components
-import { MainComponent } from './main/main.component';
-import { MenuComponent } from './menu/menu.component';
-// Icons
+import { MarkdownModule } from 'ngx-markdown';
+
 import { CancelIconComponent } from './icons/cancel/cancel-icon.component';
-import { HorizontalBarGraphIconComponent } from './icons/horizontal-bar-graph/horizontal-bar-graph-icon.component';
+import { EditIconComponent } from './icons/edit/edit-icon.component';
 import { GeomapIconComponent } from './icons/geomap/geomap-icon.component';
+import { HorizontalBarGraphIconComponent } from './icons/horizontal-bar-graph/horizontal-bar-graph-icon.component';
 import { MapOfScienceIconComponent } from './icons/map-of-science/map-of-science-icon.component';
 import { NetworkIconComponent } from './icons/network/network-icon.component';
 import { ScatterGraphIconComponent } from './icons/scatter-graph/scatter-graph-icon.component';
-import { EditIconComponent } from './icons/edit/edit-icon.component';
+import { MainComponent } from './main/main.component';
+import { MenuComponent } from './menu/menu.component';
 import { SafePipe } from './safe-pipe/safe.pipe';
 
 
+// Material
```

Comment:
Remove/format this file.

Reply:
Please remove https://github.com/cns-iu/make-a-vis/blob/0d2cd143c7745892a1de2c13d2b5ecc520784c61/projects/make-a-vis/src/lib/visualization-view/visualization-view.module.ts#L26-L28